webapp/package.json

```diff
@@ -23,5 +25,16 @@
     "simple-git": "^3.20.0",
     "yaml": "^2.3.3",
     "zod": "^3.22.4"
+  },
+  "devDependencies": {
```

Comment:
is it possible to move all dev dep to the root package.json? so we keep sub projects free from dev dependencies